README.md

```diff
@@ -36,6 +36,38 @@ JSON:
 
 The examples below omit `embeds` and `playerName` keys because they are always the same.
 
+#### Death:
+
+`%VALUELOST%` will be replaced with the price of the items you lost. If you died in PvP, `%PKER%` will be replaced with the name of your killer.
+
+**Note**: If *Distinguish PvP deaths* is disabled, all deaths will be treated as non-PvP.
```

Comment:
```suggestion
**Note**: If *Distinguish PvP deaths* is disabled, the message content will always be treated as non-PvP.
```

Technically, if distinguish is disabled, we still get `extra` with the PK info - so, can clarify we're talking about `content`